ci/compute-matrix.jq

```diff
@@ -13,29 +13,40 @@ def compute_arch($x):
 def matches($entry; $exclude):
   all($exclude | to_entries | .[]; $entry[.key] == .value);
 
-def compute_tag_prefix($x):
-  if $build_type == "pull-request" then
-    $x.IMAGE_REPO + "-" + $pr_num + "-"
-  else
-    ""
-  end |
-  $x + {TAG_PREFIX: .};
-
-def compute_used_repo($x):
-  if $build_type == "pull-request" then
+def compute_repo($x):
+  if
+    env.BUILD_TYPE == "pull-request"
+  then
     "staging"
   else
     $x.IMAGE_REPO
-  end |
-  $x + {USED_REPO: .};
+  end;
+
+def compute_tag_prefix($x):
+  if
+    env.BUILD_TYPE == "branch"
+  then
+    ""
+  else
+    $x.IMAGE_REPO + "-" + env.PR_NUM + "-"
+  end;
 
-def compute_full_prefix($x):
-  if $build_type == "pull-request" then
-    "rapidsai/"+$x.USED_REPO+":"+$x.TAG_PREFIX
+def compute_manylinux_version($x):
+  if
+    $x.LINUX_VER == "ubuntu20.04" // $x.LINUX_VER == "ubuntu18.04"
```

Comment:
this line and line 4 of this file will need to be updated as we discussed on Slack.